drkafka/src/main/java/com/pinterest/doctorkafka/KafkaClusterManager.java

```diff
@@ -943,6 +943,9 @@ private boolean checkAndReplaceDeadBrokers() {
     KafkaBroker toBeReplaced = null;
     for (Map.Entry<Integer, KafkaBroker> brokerEntry : kafkaCluster.brokers.entrySet()) {
       KafkaBroker broker = brokerEntry.getValue();
+      if (broker.isDecommissioned()) {
```

Comment:
Would it also make sense for decommissioning to actually have a some sort of a tombstone marker (this can be done via kafka topic) decommissioned brokers and remove the entry from map of brokers as well? The reason for this is the following scenario:
1. Broker is decommissioned via Dr. Kafka API
2. Dr. Kafka restarts
3. Rescans the last 12 hours of stats the broker is added back to the map and will trigger replacement

Reply:
This will be added in another PR